crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -372,12 +364,15 @@ void LateUpdateTiles()
         {
             var definitelyUnderwater = ViewerHeightAboveWater < -5f;
 
-            // TODO - dont call this every frame!! save off list from oceanbuilder?
+            var density = _material.GetVector("_DepthFogDensity");
+            var minimumFogDensity = Mathf.Min(Mathf.Min(density.x, density.y), density.z);
+            var volumeExtinctionLength = -Mathf.Log(_underwaterCullLimit) / minimumFogDensity;
+
             var tiles = OceanBuilder.OceanChunkRenderers;
             foreach (var tile in tiles)
             {
                 Renderer tileRenderer = tile.Renderer;
-                tileRenderer.enabled = (!definitelyUnderwater) || (_viewpoint.position - tileRenderer.bounds.ClosestPoint(_viewpoint.position)).magnitude < VolumeExtinctionLength;
+                tileRenderer.enabled = (!definitelyUnderwater) || (_viewpoint.position - tileRenderer.bounds.ClosestPoint(_viewpoint.position)).magnitude < volumeExtinctionLength;
```

Comment:
This always seems to be true for some reason in my most recent tests... debugging now.